mkdocs/docs/HPC/setting_up_python_virtual_environments.md

```diff
@@ -0,0 +1,221 @@
+# Python Virtual Environments (venv's)
+
+## Introduction
+
+Python virtual environments (venv's) are a way to create isolated Python environments.
+You can install packages in a venv without affecting the system Python installation.
```

Comment:
```suggestion
You can install additional Python packages in a virtual environment without affecting the system Python installation.
```

It's not just about not affecting the Python installation, but also controlling *where* additional Python packages are installed, in a controlled way (so they don't go into `.local` in your home directory, etc.)

Reply:
Applied commit and changed all other occurrences of `venv` with `virtual environment`  in [33ac8a2](https://github.com/hpcugent/vsc_user_docs/pull/639/commits/33ac8a20833fe1dcca543de078d03d72a4a89e7f)